assets/sass/modules/_consent.scss

```diff
@@ -90,5 +92,15 @@ $consent-light-text-color: #6e6e78; // Gray 600 in Odyssey
     input[data-type="save"] {
       float: right;
     }
+
+    .button-primary {
```

Comment:
why not just remove the `button-primary` class for the `Allow` button?

Reply:
That is another way, but I believe CSS overrides should be in one place so it's easier to maintain.